ops/bin/setup_clone_db.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# These scripts expect to be in `feeder.prx.org/ops/bin`
+set -e
+set -u
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+. $DIR/_env.sh
+
+echo "Dropping clone database"
+dropdb $CLONE_DB_NAME || true
+echo "Creating clone database"
+createdb $CLONE_DB_NAME
+
+echo "Running pg_restore ..."
+pg_restore --verbose -j 16 -d $CLONE_DB_NAME $LINK_FILE
+echo ""
+echo "DONE"
```

Comment:
My pg_restore is exiting with an error before it gets here:

```
...
...
pg_restore: finished item 3945 INDEX index_episodes_on_original_guid_and_podcast_id
pg_restore: finished item 4015 CONSTRAINT sessions sessions_pkey
pg_restore: finished item 3943 INDEX index_episodes_on_guid
pg_restore: launching item 4194 ACL SCHEMA public
pg_restore: creating ACL "SCHEMA public"
pg_restore: from TOC entry 4194; 0 0 ACL SCHEMA public rdsadmin
pg_restore: error: could not execute query: ERROR:  role "rdsadmin" does not exist
Command was: REVOKE ALL ON SCHEMA public FROM rdsadmin;
REVOKE USAGE ON SCHEMA public FROM PUBLIC;
GRANT ALL ON SCHEMA public TO prxapps;
GRANT ALL ON SCHEMA public TO PUBLIC;


pg_restore: finished item 4194 ACL SCHEMA public
pg_restore: finished main parallel loop
pg_restore: warning: errors ignored on restore: 3
```

Maybe needs to ignore roles or something?